frontend/src/common/MultiFileInputFileDisplay.tsx

```diff
@@ -21,7 +22,7 @@ export default function MultiFileInputFileDisplay({ file, handleDelete, onChange
 
   return (
     <Grid container spacing={1} alignItems='center'>
-      <Grid item xs={4}>
+      <Grid item>
```

Comment:
Grid items should always specify how much space they want to use. E.g. `xs={4}` means it wants to use 4 out of the 12 available columns for any screen size `xs` and above (you can also specify higher breakpoints if you want the layout to change for different screen sizes), `xs` means fill up the remaining columns or share evenly with any other grid items looking to do that same.

Aside from the file name being a Chip here and a Link in `ReleaseDispay`, we should probably make this look the same as the grid in `ReleaseDisplay` if we want to show the file size here too.